src/dashboard/src/components/administration/tests/test_usage.py

```diff
@@ -0,0 +1,53 @@
+import mock
+import os
+
+from django.conf import settings
+from django.test import TestCase
+
+from components import helpers
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestUsage(TestCase):
+    fixtures = [os.path.join(THIS_DIR, "../../../../tests/fixtures", "test_user.json")]
+
+    def setUp(self):
+        self.client.login(username="test", password="test")
+        helpers.set_setting("dashboard_uuid", "test-uuid")
+
+    def test_no_calculation(self):
```

Comment:
Nice test, the corresponding change in the dashboard works nicely with a few different values too :+1: